File: 240404/2진수로 변환하기/convert-to-binary.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] arr = new int[20];
        int count = 0;
        while(true) {
            if(n < 2) {
                arr[count++] = n;
                break;
            }
            arr[count++] = n%2;
            n /=2;
        }
        for(int i = count-1; i>=0;i--) {
            System.out.print(arr[i]);
        }
    }
}

File: 240419/2진수로 변환하기/convert-to-binary.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)

    var n = sc.nextInt()
    var cnt = 0
    var result = ""

    while(true) {
        result += n % 2

        if (n < 2) {
            break
        }

        n /= 2
    }

    print(result.reversed())
}